tlatools/org.lamport.tlatools/src/tlc2/value/impl/SetEnumValue.java

```diff
@@ -274,6 +329,8 @@ public final Value normalize() {
       return this;
     }
     catch (RuntimeException | OutOfMemoryError e) {
+      // reset isNorm since it failed
+      this.isNorm = false;
```

Comment:
this fixes un-normalized sets being classified as normalized, issue in latest main